dev-go/act/act-1.6.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-base toolchain-funcs
+
+DESCRIPTION="Automated Component Toolkit code generator"
+HOMEPAGE="https://github.com/Autodesk/AutomaticComponentToolkit"
+SRC_URI="https://github.com/Autodesk/AutomaticComponentToolkit/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/AutomaticComponentToolkit-${PV}"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RESTRICT="strip test"
+
+DOCS=( CODE_OF_CONDUCT.md CONTRIBUTING.md README.md )
```

Comment:
I don't really think users need to be aware of the `CODE_OF_CONDUCT.md`, it's more of a contributor-thing. (Same goes with your freecad).

I'd just match the default docs, and remove this line altogether from this ebuild.

Reply:
I added them in case someone wants to do the step from a user into actually helping with a project. But I'm ok, to drop these files.